include/solvers/vof.h

```diff
@@ -719,6 +737,10 @@ class VolumeOfFluid
   double       mass_first_iteration;
   double       sharpening_threshold;
 
+  double pressure_min;
+  double pressure_max;
+  double pressure_monitored_avg;
```

Comment:
Maybe put this into a small struct. PressureMonitoring or something similar. i would rather have as little floating doubles lying around.